code-challenges.js

```diff
@@ -20,37 +20,74 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 // Pseudo code:
+// Take an input number
+// Use a conditional statement to compare input numbers to temperature 212 degrees
+//  Return string statements to describe results
 
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+const boilChecker = (temp) => {
+ if (temp < 212) {
+    return `"${temp} is below boiling point"`}
```

Comment:
The backticks for sting interpolation are enough for the string. The double quotations marks "" are not required when you have the backticks.